src/transformer/common/utf8.cpp

```diff
@@ -361,4 +362,81 @@ bool normalize_string(char **_utf8Buffer, uint64_t &bufferLength)
     return true;
 }
 
+bool normalize_string(cow_string &str)
```

Comment:
the other function could be written in function of this, since they're basically the same

Reply:
The other function will be removed in the next iteration, as it'll deprecate `PWTransformer.*`, so I decided to be pragmatic.